packages/@react-spectrum/s2/chromatic/BreadcrumbsRTL.stories.tsx

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright 2024 Adobe. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+
+import {Breadcrumbs} from '../src';
+
+import type {Meta} from '@storybook/react';
+
+const meta: Meta<typeof Breadcrumbs> = {
+  component: Breadcrumbs,
+  parameters: {
+    chromaticProvider: {colorSchemes: ['dark'], backgrounds: ['base'], locales: ['ar-AE'], disableAnimations: true}
+  },
+  title: 'S2 Chromatic/BreadcrumbsRTL'
+};
+
+export default meta;
+export {Dynamic, Powerset} from './Breadcrumbs.stories';
```

Comment:
Ah this is because I've set
```
Dynamic.parameters = {
  // TODO: add ar-AE when we get intl strings
  chromaticProvider: {colorSchemes: ['light'], backgrounds: ['base'], locales: ['en-US'], disableAnimations: true}
};
```
In the Breadcrumb stories that they use to make sure `ar-AE` doesn't actually apply since it blows up the story due to missing intl strings. When those strings are added I'll update these, will add a comment